dangerfile.js

```diff
@@ -36,16 +36,22 @@ function generateMDTable(headers, body) {
  * Generates a user-readable string from a percentage change
  * @param {string[]} headers
  */
-function emojiPercent(change) {
-  if (change > 0) {
+function addPercent(change, includeEmoji) {
+  if (change > 0 && includeEmoji) {
     return `:small_red_triangle:+${change}%`;
-  } else if (change <= 0) {
+  } else if (change > 0) {
+    return `+${change}%`;
+  } else {
     return `${change}%`;
   }
 }
 
-function boldRow(row) {
-  return row.map(element => `**${element}**`);
+function setBoldness(row, isProd) {
```

Comment:
Nit: this could be called `isBold` because this particular function cares about the formatting rather than *when* we make it bold